app/workers/fieri_notify_worker.rb

```diff
@@ -21,9 +21,25 @@ def perform(cookbook_version_id)
     data = {
       'cookbook_name' => cookbook_version.name,
       'cookbook_version' => cookbook_version.version,
-      'cookbook_artifact_url' => cookbook_version.tarball.url
+      'cookbook_artifact_url' => cookbook_artifact_url(cookbook_version)
     }
 
     response = Net::HTTP.post_form(uri, data)
   end
+
+  private
+
+  def cookbook_artifact_url(cookbook_version)
```

Comment:
Does this method also need to handle [the case of S3 with expiring URLs](https://github.com/chef/supermarket/blob/master/app/controllers/cookbook_versions_controller.rb#L37-L43)? I only just found that. Some cursory reading on Paperclip doesn't surface a good abstraction for the attachment. :disappointed: